backups/cognito/backup_users.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import csv
+from datetime import datetime
+from botocore.exceptions import ClientError
+import os
+
+client = boto3.client('cognito-idp')
+s3_client = boto3.client('s3')
+
+def backup_users():
+    try:
+        all_users = get_all_users()
+    except Exception as e:
+        print('Getting current cognito users has failed ', e)
+
+    csv_headers = ['cognito:username','name','given_name',
```

Comment:
we need to pass a value to every field even if its empty - see https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-using-import-tool-csv-header.html